<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlickrSearchMainWindow</class>
 <widget class="QMainWindow" name="FlickrSearchMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Flickr Image Search - Search Images from Flickr</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="grpFlickrSearch">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridFlickrSearch">
         <item row="2" column="1">
          <widget class="QLineEdit" name="edtFlickrSearch">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>22</pointsize>
            </font>
           </property>
           <property name="placeholderText">
            <string>Search Images on Flickr</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lblFlickrSearch">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../resources/resources.qrc">:/icons/icons/FlickrImageSearch.png</pixmap>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QPushButton" name="btnFlickrSearch">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>22</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QListWidget" name="listWidget">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="iconSize">
       <size>
        <width>128</width>
        <height>128</height>
       </size>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideMiddle</enum>
      </property>
      <property name="flow">
       <enum>QListView::TopToBottom</enum>
      </property>
      <property name="resizeMode">
       <enum>QListView::Adjust</enum>
      </property>
      <property name="layoutMode">
       <enum>QListView::SinglePass</enum>
      </property>
      <property name="viewMode">
       <enum>QListView::IconMode</enum>
      </property>
      <property name="uniformItemSizes">
       <bool>true</bool>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="itemAlignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="grpPageTrack">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="12">
        <widget class="QLineEdit" name="edtNextPage">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnFirstPage">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>First Page</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="btnPreviousPage">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Prev Page</string>
         </property>
        </widget>
       </item>
       <item row="0" column="11">
        <widget class="QPushButton" name="btnNextPage">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Next Page</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="edtTotalPages">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QLabel" name="lblCurrentPage">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Current Page:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="14">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="lblTotalPages">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Total Pages:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="13">
        <widget class="QPushButton" name="btnLastPage">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Last Page</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <widget class="QLineEdit" name="edtPreviousPage">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="8">
        <widget class="QLineEdit" name="edtCurrentPage">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
